src/development/variables.md

```diff
@@ -304,7 +309,7 @@ Certain variable name prefixes have special meaning.  A few of these are defined
 
 ### Top-level environment variables
 
-By default, project and environment variables are only added as part of the `$PLATFORM_VARIABLES` Unix environment variable.  However, you can also expose a variable as its own Unix environment variable by giving it the prefix `env:`.  
+By default, project and environment variables are only added as part of the `$PLATFORM_VARIABLES` Unix environment variable.  However, you can also expose a variable as its own Unix environment variable by giving it the prefix `env:`.
```

Comment:
Please, don't forget to mention that will became uppercase if they put the prefix `env:`

Reply:
That should be done but it's a separate fix from this one.  (This is just a whitespace fixup from my IDE, not part of this patch.)  Should we include it in this PR or make it separate?